Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py

```diff
@@ -435,6 +440,16 @@ def delete_profile(client: Client, **args) -> str:
     return results
 
 
+def query_search(client, **args) -> CommandResults:
+    params = {"query": args.get("query")}
+
+    r = client._http_request("GET", "search", params=params)
+
+    results = CommandResults(
+        outputs_prefix="Triage.samples", outputs_key_field="id", outputs=r["data"]
```

Comment:
The output_prefix should be the same as the output in the yml file
So either change the output in the yml file or change here to `Triage.submissions`